sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -28,7 +29,12 @@
 
 class ContainerRegistryClient(ContainerRegistryBaseClient):
     def __init__(
-        self, endpoint: str, credential: "Optional['AsyncTokenCredential']" = None, *, audience: str, **kwargs: "Any"
+        self,
+        endpoint: str,
+        credential: "Optional['AsyncTokenCredential']"=None,
```

Comment:
```suggestion
        credential: Optional["AsyncTokenCredential"] = None,
```
I think we only need quotes around the credential type.

Reply:
Updated.